ui/src/app/app.module.ts

```diff
@@ -84,7 +84,7 @@ import {PublishCollectionComponent} from "./collection/detail/publish-collection
 import {BlockingLoaderComponent} from "./core/blocking-loader.component"
 import {CollectionSkillSearchComponent} from "./collection/collection-skill-search.component"
 import {BatchImportComponent} from "./richskill/import/batch-import.component"
-import {FieldMappingSelectComponent, FieldMappingTableComponent} from "./richskill/import/field-mapping-table.component"
+import {FieldMappingSelectComponent, FieldMappingTableComponent, BatchImportDestinationSelectComponent} from "./richskill/import/field-mapping-table.component"
```

Comment:
i think we should line break each one, when there is more than one import from a file. Just like lines 88-90